### PR TITLE
Trends Page: Add labels

### PR DESCRIPTION
**Pull request by DarkFlamingo**

### CODE REVIEW

**Comment by DarkFlamingo** [comment]
image

File: frontend/src/scenes/TrendsPage/components/ProjectsChart/ChartLabels/Label/index.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+const Label: FC<LabelProps> = ({ project, color }) => (
+  <Col className={classnames(classes.item, 'justify-content-center')}>
+    <div className={classes['color-wrapper']}>
+      <div style={{ background: color }} className={classes['color-box']} />
+    </div>
+    <div className={classes['text-wrapper']}>
+      <p className={classes.name}>{сutWord(project.name, 22)}</p>
+      <p className={classes.data}>{`${project.views} views`}</p>
```
Comment (yevhenii-moroziuk) [inline frontend/src/scenes/TrendsPage/components/ProjectsChart/ChartLabels/Label/index.tsx]: Do not forget about localization

File: frontend/src/scenes/TrendsPage/components/ProjectsChart/ChartLabels/Label/index.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  color: string;
+}
+
+const Label: FC<LabelProps> = ({ project, color }) => (
+  <Col className={classnames(classes.item, 'justify-content-center')}>
+    <div className={classes['color-wrapper']}>
+      <div style={{ background: color }} className={classes['color-box']} />
+    </div>
+    <div className={classes['text-wrapper']}>
+      <p className={classes.name}>{сutWord(project.name, 22)}</p>
```
Comment (yevhenii-moroziuk) [inline frontend/src/scenes/TrendsPage/components/ProjectsChart/ChartLabels/Label/index.tsx]: Use constant

File: frontend/src/scenes/TrendsPage/options/ProjectsOptions.ts [hunk 3]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```
Comment (yevhenii-moroziuk) [inline frontend/src/scenes/TrendsPage/options/ProjectsOptions.ts]: Why do you need to use eslint-disable?

**Review by yevhenii-moroziuk — CHANGES_REQUESTED** [review]

File: frontend/src/scenes/TrendsPage/options/ProjectsOptions.ts (hunk 3 above)
Comment (DarkFlamingo) [reply frontend/src/scenes/TrendsPage/options/ProjectsOptions.ts]: I use it because I have unused parameters in the function

Comment (yevhenii-moroziuk) [reply frontend/src/scenes/TrendsPage/options/ProjectsOptions.ts]: u can use prefix _

Comment (DarkFlamingo) [reply frontend/src/scenes/TrendsPage/options/ProjectsOptions.ts]: Thank you very much I do not know about such opportunity

File: frontend/src/scenes/TrendsPage/components/ProjectsChart/ChartLabels/Label/index.tsx (hunk 2 above)
Comment (DarkFlamingo) [reply frontend/src/scenes/TrendsPage/components/ProjectsChart/ChartLabels/Label/index.tsx]: For color ?

Comment (DarkFlamingo) [reply frontend/src/scenes/TrendsPage/components/ProjectsChart/ChartLabels/Label/index.tsx]: I give this component colors that I take from the helper

Comment (yevhenii-moroziuk) [reply frontend/src/scenes/TrendsPage/components/ProjectsChart/ChartLabels/Label/index.tsx]: for magic number 22

**Review by yevhenii-moroziuk — APPROVED** [review]

**Pull request merged**